content/kubernetes/re-databases/set-up-ingress-controller.md

```diff
@@ -88,9 +88,12 @@ Install one of the supported ingress controllers:
         http:
           paths:
           - path: /
+            pathType: Prefix
```

Comment:
On one hand, the operator is creating the ActiveActive ingresses with `pathType: ImplementationSpecific` so it would be nice to be on par with it. And anyway's Nginx's behavior for `ImplementationSpecific` is identical to `Prefix` and HAProxy's behavior is quite close too (`path-type: begin`).

On the other hand, `Prefix` is somewhat better match semantically, especially if this inspires the user how to create similar ingress resources for the cluster API/UI which are actually HTTP.

On yet another hand, this service is TCP-based so this hardly even matters.

/shrug

Reply:
Bottom line, I suggest to stick to `pathType: ImplementationSpecific`.

Reply:
I will change this to ImplementationSpecific.